deps/checksums/curl

```diff
@@ -1,36 +1,34 @@
-LibCURL-fd8af649b38ae20c3ff7f5dca53753512ca00376.tar.gz/md5/f082283e6a35fcba5b63c9a6219d8003
-LibCURL-fd8af649b38ae20c3ff7f5dca53753512ca00376.tar.gz/sha512/3bea5fa3fb6d29651daa923ae6bcb8eeb356ab9f2a1f3e005a6b746b617b0cf609aed4cadda4181783959840873c04b18e34e45ab973549169d19775a05ea01e
```

Comment:
The new file is missing these two lines, and the failure at the end of the build job I guess is related to this.  But I don't know how these two checksums should come from.

Reply:
I wondered what to do with `stdlib/LibCURL.version`, but I have how idea how it relates to the rest.

Reply:
Oh, `LibCURL.jl` is the standard libraries whose source code is pulled from an external repository: https://github.com/JuliaLang/julia/blob/86f5501f248bb30304fc19a9abc8affee98e42f8/stdlib/LibCURL.version#L1-L4 I guess you may need to simply restore these two lines then.